### PR TITLE
STR-420 - add chart templates for sample project

### PR DESCRIPTION
**Pull request by MihelisAndreas**
Nu sunt sigur de cateva inlocuiri cu application name in values -TierName,NodeName,NodePrefix , imaginea ramane chassis/{{applicationName}} ?

### CODE REVIEW

File: initializr-generator/src/main/resources/templates/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ENV JAVA_AGENT -javaagent:/opt/appdynamics/javaagent.jar
+
+ENTRYPOINT [ "sh", "-c", \
+    "java -Djava.security.egd=file:/dev/./urandom ${APPDYNAMICS_CONTROLLER_HOST_NAME:+$JAVA_AGENT} ${APPDYNAMICS_NODE_PREFIX:+-Dappdynamics.agent.reuse.nodeName=true -Dappdynamics.agent.reuse.nodeName.prefix=$APPDYNAMICS_NODE_PREFIX} -jar /app.jar" \
+    ]
+# everything above is not impacted by a new build and will be cached by Docker
+
+ARG JAR_FILE
+ADD ${JAR_FILE} app.jar
+
```
Comment (coman-darie) [inline initializr-generator/src/main/resources/templates/Dockerfile]: Please don't forget to use the docker file in the same form as we discussed (commenting out the Dockerfile section for AppDynamics integration).

Comment (coman-darie) [inline initializr-generator/src/main/resources/templates/Dockerfile]: Please don't forget to use the docker file in the same form as we discussed (commenting out the Dockerfile section for AppDynamics integration).

File: initializr-generator/src/main/resources/templates/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+FROM registry.misys.global.ad/ffdc-base/alpine-java:oraclejre-8
+ARG JAR_FILE="target/{{applicationName}}"
```
Comment (coman-darie) [inline initializr-generator/src/main/resources/templates/Dockerfile]: By default, the `{{applicationName}}` will be replaced here with `ChassisStarterApplication`. In turn, after building the generated project, the jar created in the `target` folder has `chassis-starter-0.0.1-SNAPSHOT.jar`.
Therefore, this Dockerfile cannot be used in this form.
Solution: 
- I would add `<finalName>${project.name}</finalName>` in the pom file for the generated project. By default, the `${project.name}` is `chassis-starter`
- In the Dockerfile I would add `{{artifact}}` which should be replaced by the Artifact value which users fill in

File: initializr-generator/src/main/resources/templates/Dockerfile (hunk 1 above)
Comment (coman-darie) [inline initializr-generator/src/main/resources/templates/Dockerfile]: Please don't forget to use the docker file in the same form as we discussed (commenting out the Dockerfile section for AppDynamics integration).

File: initializr-generator/src/main/resources/templates/values.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+global:
+  Ingress:
```
Comment (MihelisAndreas) [inline initializr-generator/src/main/resources/templates/values.yaml]: should delete this ?

**Pull request merged**